packages/abstract-signer/src.ts/index.ts

```diff
@@ -23,6 +23,7 @@ import {
 } from "@hashgraph/sdk";
 import * as Long from "long";
 import { SignedTransaction, TransactionBody } from "@hashgraph/proto";
+import { splitInChunks } from "@ethersproject/strings";
```

Comment:
I think that `splitInChunks` is better to be in the `bytes` package and to use only `bytes` instead of `string`.